docs/testbed/README.testbed.Setup.md

```diff
@@ -195,28 +220,24 @@ Once you are in the docker container, you need to modify the testbed configurati
     ansible_user: admin
     EOT
     ```
-
-- PTF Docker
-    - Update the docker registry information in [`vars/docker_registry.yml`](/ansible/vars/docker_registry.yml).
-
-## Setup VMs on the Server
-
-1. Start the VMs:
-    ```
-    # /data/sonic-mgmt/ansible
-    ./testbed-cli.sh start-vms server_1 password.txt
-    ```
-    Please note: `password.txt` is the ansible vault password file name/path. Ansible allows users to use `ansible-vault` to encrypt password files. By default, this shell script **requires** a password file. If you are not using `ansible-vault`, just create an empty file and pass the file name to the command line. **The file name and location is created and maintained by the user.**
+    -update the cEOS vars in [`ansible/group_vars/all/ceos.ymal`](/ansible/group_vars/all/ceos.yml).
     ```
-    echo "" > ./password.txt
+    ceos_image_filename: cEOS64-lab-4.25.10M.tar
+    ceos_image_orig: ceosimage:4.25.10M
+    ceos_image: ceosimage:4.25.10M
+    skip_ceos_image_downloading: true
     ```
-
-2. Check that all the VMs are up and running:
+    **NOTE: We are using local ceos image, hence the skip ceos image downloading should be set as true.
```

Comment:
Using local docker registry is a good alternative. However, I suggest documenting it as an alternative to the Microsoft registry. Currently the document sounds like using local registry is the only method.

Reply:
OK, I will clarify this ambiguity.